ui/src/components/giving/GivingPage.js

```diff
@@ -0,0 +1,236 @@
+      </VStack>
+      <Center>
+        <Flex
+          h={["100%", "100%", "13em"]}
+          direction={["column", "column", "row"]}
+          paddingBottom="2"
+          minW="100%"
+        >
+          <Box w={["100%", "100%", "32%"]} h="20%" borderRadius="lg">
+            <Heading as="h3" size="xl" paddingBottom="1vw" color="#319795">
+              Ways to Give:
+            </Heading>
+            <Text fontWeight="bold" paddingBottom="1vw">
+              There are a few different ways you can give to our church
+            </Text>
+            <Text as="i" color="#319795">
+              *When giving, please always use your legal name and provide the
+              same email address consistently.
+            </Text>
+          </Box>
+          <Spacer />
+          <GivingCard
+            text="FPS"
+            imageLink={process.env.PUBLIC_URL + "/giving/FPS.png"}
+          />
+          <Spacer />
+          <GivingCard
+            text="Bank Transfer"
+            imageLink={process.env.PUBLIC_URL + "/giving/Transfer.png"}
+          />
+        </Flex>
+      </Center>
+
+      <Center paddingTop="1vw">
+        <Flex
+          direction={["column", "column", "row"]}
+          minW="100%"
+          h={["100%", "100%", "13em"]}
+        >
+          <GivingCard
+            text="Cash"
+            imageLink={process.env.PUBLIC_URL + "/giving/Cash.png"}
+          />
+          <Spacer />
+          <GivingCard
+            text="Online Giving"
+            imageLink={process.env.PUBLIC_URL + "/giving/Online.png"}
+          />
+          <Spacer />
+          <GivingCard
+            text="Cheque"
+            imageLink={process.env.PUBLIC_URL + "/giving/Cheque.png"}
+          />
+        </Flex>
+      </Center>
+      <Box paddingTop="2vh">
+        <Text as="i">
+          Personal information is kept confidential, used only for tax receipt
+          purposes, and is only accessible by the Stewardship Team.
+        </Text>
+      </Box>
+      <Box paddingTop="2vh">
+        <Text as="i">
+          If you have any questions, please do not hesitate to contact us:&nbsp;
+        </Text>
+        <Text as="i" fontWeight="bold">
+          stewardship@hongkong.hmcc.net
+        </Text>
+      </Box>
+      <Box paddingBottom="10vh">
+        <Box display={{ base: "none", lg: "block" }}>
+          <Heading as="h1" fontSize={["5xl", "5xl", "6xl"]} py="4vw" px="1vh">
+            Frequently Asked Question
+          </Heading>
+        </Box>
+        <Box display={{ base: "block", lg: "none" }}>
+          <Heading as="h1" fontSize={["5xl", "5xl", "6xl"]} py="4vw" px="1vh">
+            FAQs
+          </Heading>
+        </Box>
+
+        <Accordion allowMultiple>
+          <AccordionItem>
+            <h2>
+              <AccordionButton>
+                <Box flex="1" textAlign="left">
+                  <Heading as="h4" size="md" py="4" minw="100%">
+                    How does registration work?
```

Comment:
This is more of a content question but, why is the FAQ contents completely unrelated to Giving itself? Shouldn't there be giving specific FAQ? haha